website/docs/events/m1.md

```diff
@@ -40,7 +42,125 @@ Please consider topics such as:
 
 During the workshop, participants will put their ideas into practice and take the first steps towards having a full-fledged class that uses modern technology for efficient teaching.
 
-## Open Discussion over Lunch: 13:00 - 14:00
+### Open Discussion over Lunch: 13:00 - 14:00
 
 Lunch is on the Open Education Hub team.
 As always, lunch will be a pretext for more open-ended discussions regarding the Open Education Hub project and the guests' further involvement with it. 
+
+## Summary
+
+Find the entire discussion from our meeting below in text format or on [YouTube](https://www.youtube.com/watch?v=RkUnCXL_BtA&ab_channel=OpenEducationHub) in audio format (Romanian only).
+
+### Project Overview
+
+RD: We want to see how realistic is our idea.
+RD: Our content is often on the Wiki, but is not easily reviewed / updated. I had problems. When suggestions arise, people send the fix to you and you have to deal with it. We want to fix this.
+RD: We don't have a collaborative way to do it.
+RD: Folks from Singapore are using our content as it is, they don't have a wat to update it for their needs.
+RD: A good example on how it's supposed to be is the Operating Systems 2 repository. It's a repo based on the Linux Kernel repository that has our deploy infrastructure on top. Everything works in a VM, it is easy to use, update, create snapshots. Students can use it as it is, can come with fixes via pull requests.
+RD: This was our motivation.
+RD: We usualy receive feedback from students that we have outdated content, information that is repetitive or even mistakes. They do not have write access to OCW and they cannot be granted write access because OCW does not have a method for reviewing materials.
+RD: Usually content is private or, if it is publice, it's very difficult tu update.
+RD: It's not easy to create super high quality content, you need a lot of iterations.
+RD: We want to have as many actions as possible automated, everything should be in one place: in a git repository. We want us, teachers, to spend more time on mentoring students, rather than doing repetitive tasks that can be easily automated. Spending time on repetitive tasks makes us lose time, focus and will to educate students.
+RD: Usually content is difficult to update, it is non-digitalized, there is no such thing as versioning, it's difficult to distribute. [Github](https://github.com/) makes it easy to contribute, easy to review, easy to report problems, easy to reuse.
+RD: We use a lot of technologies such as Markdown, reveal.md, draw.io, git, ffmpeg.
+RD: We want the evaluations process to be fully automated. We want to provide an infrastructure where students can practice.
+AO: Where is the Linux Kernel Labs repository?
+RD: Here it is, on [GitHub](https://github.com/linux-kernel-labs).
+Somebody (unknown voice): How much does your work so far depends on the fact that the context is in text format?
```

Comment:
```suggestion
Somebody (unknown voice): How much does your work so far depend on the fact that the context is in text format?
```
This question doesn't make sense. What do you mean by "the context is in text format"?

Reply:
I meant `content` there.